src/buildMiddleware.ts

```diff
@@ -37,11 +37,13 @@ export function buildMiddleware<Definitions extends EndpointDefinitions, Reducer
       const invalidateEntities = calculateProvidedBy(
         endpointDefinitions[action.meta.arg.endpoint].invalidates,
         action.payload,
-        state.mutations[action.meta.arg.endpoint]?.[action.meta.requestId]?.arg
+        action.meta.arg.arg
+        // state.mutations[action.meta.arg.endpoint]?.[action.meta.requestId]?.arg,
```

Comment:
I think this might only work when tracked is enabled? But either way, this should just reference the arg from the fulfilled action.meta, right?

Reply:
very good point.